openfecwebapp/__init__.py

```diff
@@ -100,6 +100,9 @@ def get_election_url(candidate, cycle, district=None):
 def asset_for(path):
     return url_for('static', filename=assets[path].lstrip('/'))
 
+def datatable_script(slug):
```

Comment:
This doesn't seem to include logic that's specific to data tables. How about calling it something like `asset_for_page`?